templates/mod.html

```diff
@@ -75,7 +75,7 @@ <h1 title="{{ mod.name }}">
         <div class="{% if user %}col-md-2{% else %}col-md-4{% endif %}">
             <a  class="btn btn-block btn-lg btn-primary piwik_download" id="download-link-primary"
                 href="{{ url_for("mods.download", mod_id=mod.id, mod_name=mod.name, version=latest.friendly_version) }}">
-                Download</a>
+                Download {% if size_versions[latest.id] != None %} ({{ size_versions[latest.id] }}) {% endif %}</a>
```

Comment:
Never ever compare to `None` with `==` or `!=` :upside_down_face: 
```suggestion
                Download {% if size_versions[latest.id] is not None %} ({{ size_versions[latest.id] }}) {% endif %}</a>
```